src/lib.rs

```diff
@@ -1939,6 +1957,17 @@ impl RtcConfig {
         self.crypto_provider
     }
 
+    /// Returns the configured DTLS certificate configuration.
```

Comment:
Let's flesh this out a bit. Say what the default is.

Reply:
Done